src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.ts

```diff
@@ -41,6 +58,20 @@ export class DotEditLayoutComponent implements OnInit {
 
         this.route.data.pluck('pageView').subscribe((pageView: DotPageView) => {
             this.pageView = pageView;
+            console.log('pageView', pageView);
+            this.form.setValue({
+                title: this.isLayout() ? null : this.pageView.template.title,
+                layout: {
+                    body: this.pageView.layout.body || {},
+                    header: this.pageView.layout.header,
+                    footer: this.pageView.layout.footer,
+                    sidebar: {
```

Comment:
We should match this exactly with what it came from the server, isn’t?

Reply:
maybe later when we'll clean the PageView object